boa/src/object/mod.rs

```diff
@@ -961,6 +966,64 @@ impl<'context> ConstructorBuilder<'context> {
         self
     }
 
+    /// Add new accessor property to the constructor's prototype.
+    #[inline]
+    pub fn accessor_property<K>(
```

Comment:
Maybe also rename `.accessor_property()` to `.accessor()` since accessors are already properties so I don't think it's ambiguous.